setup.cfg

```diff
@@ -84,3 +88,7 @@ python_files=*.py
 
 # Any function in the test file is a test
 python_functions=test_*
+
+[tool:isort]
+# Ensure isort is black compatible
+profile = black
```

Comment:
Not really related to this PR, maybe it should be pushed straight to master? It's needed to avoid conflicts between `black` and `isort` e.g. for the pre-commit checks

Reply:
This will be merged soon, does it make a difference?